tests/system/providers/amazon/aws/example_batch.py

```diff
@@ -15,6 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 from datetime import datetime
+from typing import List, Optional, Tuple
```

Comment:
Can you try using `__future__.annotations` like in PR: https://github.com/apache/airflow/pull/26290/files

Reply:
I think this might be better done after I merge it (today :))

Reply:
Done